schema/artwork/utilities.js

```diff
@@ -19,3 +21,25 @@ export const isTooBig = ({ width, height, metric }) => {
     parseFloat(height) > LIMIT[metric]
   );
 };
+
+export const isEmbeddedVideo = ({ website, category }) => (
+  website.match('vimeo|youtu') &&
+  category.match('Video')
+);
+
+export const embed = (website, { width, height, autoplay }) => {
+  const { host } = parse(website);
+  const id = website.split('/').pop();
+
+  switch (host) {
+    case 'youtu.be':
+    case 'youtube.com':
+      return `<iframe width='${width}' height='${height}' src='https://www.youtube.com/embed/${id}?rel=0&amp;showinfo=0&amp;autoplay=${autoplay ? 1 : 0}' frameborder='0' allowfullscreen></iframe>`;  // eslint-disable-line max-len
+
+    case 'vimeo.com':
+      return `<iframe width='${width}' height='${height}' src='//player.vimeo.com/video/${id}?color=ffffff&amp;autoplay=${autoplay ? 1 : 0}' frameborder='0' allowfullscreen></iframe>`; // eslint-disable-line max-len
+
+    default:
+      return null;
+  }
```

Comment:
Adapted from https://github.com/artsy/force/blob/dc19927e1476b56c57c49f7bb38094a849d62bf2/apps/artwork/client/video.coffee#L26-L4

This could probably be a more generic thing, but it's use is limited ATM.